docs/docs/self-hosting/manage/updating_scaling.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Updating and Scaling
+---
+
+This page shows, how you can improve your day-two operations
+by separating the init and setup phases from the ZITADEL start runtime process.
+
+## Scaling ZITADEL
+
+For production setups, we recommend that you run  ZITADEL in a highly available mode.
+As all shared state is managed in the database,
+the ZITADEL binary itself is stateless.
+You can easily run as many ZITADEL binaries in parallel as you want.
+If you use [Knative](/docs/self-hosting/deploy/knative)
+or [Google Cloud Run](https://cloud.google.com/run) (which uses Knative, too),
+you can even scale to zero.
+Especially if you use an autoscaler that scales to zero,
+it is crucial that you minimise startup times.
+
+## Updating ZITADEL
+
+When updating ZITADEL, the setup job takes care of database migrations.
```

Comment:
Is the setup job `zitadel setup`? I'm not completely sure if I would have to run that command manually or if this is done automatically when using a new version (I assume manual).

Reply:
I added a TL;DR in the beginning where I tried to make that clear